packages-tests/Parallel/Command/WorkerCommandLineFactoryTest.php

```diff
@@ -55,6 +55,7 @@ public function test(array $inputParameters, string $expectedCommand): void
             2000
         );
 
+        $workerCommandLine = str_replace("'main_script' worker", "'main_script' '' worker", $workerCommandLine);
```

Comment:
Could you add a notes here that it's macos related? Just to avoid false removal in the future. 
Same for the one above.

Reply:
Alternatively add a macos github action job

Reply:
@staabm That would be great too, but comment it better for warning in case of local cleanup. CI should only verify edge cases, not inform about them.

Reply:
sure, I created PR:

- https://github.com/rectorphp/rector-src/pull/4945